src/excmds.ts

```diff
@@ -2331,36 +2331,27 @@ export async function tabclose(...indexes: string[]) {
     return done
 }
 
-/** Close all tabs to the right of the current one
+/** Close all tabs to the side specified
  *
  */
 //#background
-export async function tabclosealltoright() {
+export async function tabcloseallto(side) {
```

Comment:
```suggestion
export async function tabcloseallto(side: string) {
```

You need to give this a type so our python preprocessor knows what to convert its arguments to.

Reply:
Thank you, I saw it in the tests details but will this modification be enough ? I didn't reviewed all the code but i assume the tabcloseall functions are called somewhere else.